src/Infrastructure/Controller/Map/Fragments/MapSearchController.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Infrastructure\Controller\Map\Fragments;
+
+use App\Application\MapGeocoderInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+use Symfony\Component\Routing\Annotation\Route;
+
+final class MapSearchController
+{
+    public function __construct(
+        private \Twig\Environment $twig,
+        private MapGeocoderInterface $mapGeocoder,
+    ) {
+    }
+
+    #[Route(
+        '/carte/search',
+        name: 'fragment_carto_search',
+        methods: ['GET'],
+    )]
+    public function __invoke(Request $request): Response
```

Comment:
```suggestion
    public function __invoke(#[MapQueryParameter] string $search ): Response
```

Depuis Symfony 6.3 on peut utiliser directement l'attribut `MapQueryParameter`.